doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
In my PR it was just a renaming, you could avoid breaking compatibility by just adding:
```
def CoordSysCartesian(arg1, *args, **kwargs):
    return CoordSys3D(arg1, new_arg, *args, **kwargs)
```